src/js/components/HostTable.js

```diff
@@ -5,6 +5,7 @@ import {StoreMixin} from 'mesosphere-shared-reactjs';
 import {Tooltip} from 'reactjs-components';
 
 var HostTableHeaderLabels = require('../constants/HostTableHeaderLabels');
+import IconServers from '../../img/components/icons/icon-medium/servers.svg?name=IconServers';
```

Comment:
It's actually a Webpack loader that takes the SVG and returns a React component. It's this one here: https://github.com/jhamlet/svg-react-loader